src/mlpack/methods/ann/ffn.hpp

```diff
@@ -142,6 +142,16 @@ class FFN
   void Predict(arma::mat& predictors, arma::mat& results);
 
   /**
+   * Predict the responses to a given set of predictors. The responses will
+   * reflect the output of the given output layer as returned by the
+   * output layer function.
+   *
+   * @param predictors Input predictors.
+   * @return Output predictions of responses into.
+   */
+  arma::mat Predict(arma::mat predictors);
+
```

Comment:
Oh. The reason why I didn't use `arma::mat Predict(arma::mat& predictors);` is that I want `predictors` to stay unchanged after calling `Predict`. So I want a copy here. If the signature of the original function is `void Predict(const arma::mat& predictors, arma::mat& results);`, I won't write this wrapper. But I find it's not so easy to change the original function to const reference.